### PR TITLE
VGNWB-79: Added the new page home.html showing the required information

### PR DESCRIPTION
**Pull request by apchandak24**
The code is not yet complete, need to find way to get the relation count

### CODE REVIEW

**Comment by kardanitin** [comment]
@erickpeirson : the code is yet not complete for this story. We started making changes on monday and wanted to show you the a working POC for the story. We will continue to refine it and complete it by wednesday.

File: annotations/views.py [hunk 1]
```diff
@@ -51,9 +51,19 @@
...
+    user_count = VogonUser.objects.filter(is_active=True).count()
+    text_count = Text.objects.all().count()
+    #relation_count = Relation.all().count()
+    context = RequestContext(request, {
+        'user_count': user_count,
+        'text_count': text_count,
+        'relation_count':0
+    })
+    return HttpResponse(template.render(context))
+    # if request.user.is_authenticated():
```
Comment (erickpeirson) [inline annotations/views.py]: Can you please delete these commented lines, if they are no longer needed?

File: vogon/urls.py [hunk 2]
```diff
@@ -42,6 +42,7 @@
 remotecollection_router.register('resource', views.RemoteResourceViewSet, base_name='collection')
 
 urlpatterns = [
+    url(r'^home/$',views.home,name="home"),
```
Comment (erickpeirson) [inline vogon/urls.py]: Always leave a space after each comma.

File: annotations/views.py [hunk 3]
```diff
@@ -51,9 +51,19 @@
 
 
 def home(request):
```
Comment (erickpeirson) [inline annotations/views.py:53]: Can you write a short docstring describing the view?

**Comment by erickpeirson** [comment]
@apchandak24 Great! Can you correct the few minor things that I pointed out above? I'll leave the PR open -- commit and push your changes, and it will automatically be updated.

**Comment by erickpeirson** [comment]
Thanks guys, good work.

**Pull request merged**